src/MCPClient/tests/test_json_conversion.py

```diff
@@ -7,8 +7,12 @@
 JSON_MULTICOLUMN = '[{"filename": "objects/test.txt", "dc.subject": ["foo", "bar", "baz"], "dc.title": "This is a test item"}]'
 CSV_MULTICOLUMN = 'filename,dc.subject,dc.subject,dc.subject,dc.title\r\nobjects/test.txt,foo,bar,baz,This is a test item\r\n'
 
+JSON_NULL = '[{"dc.title": "This is a test item", "filename": "objects/test.txt", "spurious metadata": null}]'
+CSV_NULL = 'filename,dc.title,spurious metadata\r\nobjects/test.txt,This is a test item,\r\n'
+
```

Comment:
If you could update or add a new test case with a `null` value inside a list, that would be great.